exercises/hello-world/description.md

```diff
@@ -1,46 +1,11 @@
 ["Hello, World!"](http://en.wikipedia.org/wiki/%22Hello,_world!%22_program) is
-the traditional first program for beginning programming in a new language.
+the traditional first program for beginning programming in a new language
+or environment.
 
-**Note:** You can skip this exercise by running:
+The objectives are simple:
 
-    exercism skip $TRACK_ID hello-world
+- Write a function that says "Hello, World!".
+- Run the tests and make sure that they succeed.
+- Submit your solution and check it at the website.
 
-## Specification
-
-Write a `Hello World!` function that can greet someone given their name.  The
-function should return the appropriate greeting.
-
-For an input of "Alice", the response should be "Hello, Alice!".
-
-If a name is not given, the response should be "Hello, World!"
-
-## Test-Driven Development
```

Comment:
I think that the instructions where misplaced.

Not all exercises implemented the `hello-world` exercise. Moving the instructions to `leap`'s `description.md` would just reproduce the problem in another place.

The immediate solution I can think of is track-dependent: Move it to the `HINTS.md` from the first exercise that is not `hello-world`. This is more flexible than forcing it to be `leap`.

This way, each track can customize the instructions to reflect what are the best practices for testing in the language. Not all languages are so TDD focused as `ruby` seems to be, specially the strongly-typed ones, that force users to type-check everything before running a single test.

I just create a Gist of a `HINTS.md` file with the old instructions [here](https://gist.github.com/anonymous/dc6d262de44db71441d8da16c41c4c80), to make migration easier. If you think this is a good idea, we could link it in a more visible place to help the track maintainers.